processor/processor_group.py

```diff
@@ -0,0 +1,15 @@
+from collections.abc import Sequence
+from processor.context import Context
+from processor.processor import AbstractProcessor
+
```

Comment:
Suggest to rename file to `group.py`, word `processor` already in folder/namespace name.